hicommonwealth/commonwealth
# Added iOS social login

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6509

## Description of Changes
- Adds iOS/apple login in the new AuthModal -> "Email or social" tabs

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_NEW_SIGN_IN_MODAL` in `.env`
- Logout, visit any page in the app and open AuthModal -> "Email or social" tabs
- Click on the "Apple" button and verify that it logs you in correctly.


## Deployment Plan
N/A

## Other Considerations
N/A
